src/components/companion/blueprintProfiles/extraInfo.tsx

```diff
@@ -0,0 +1,90 @@
+import { FC } from 'react';
+import AdditionalLinks from './additionalLinks';
+import { CompanionButton } from '../../ui/companion-button';
+import { AnimatedBorder } from '../../ui/animated-border';
+
+
+interface UserProfile {
+    name: string;
+    linkedIn: string;
+    funFacts: string[];
+    interests: string[];
+    additionalLinks: string[];
+}
+
+interface ExtraInfoProps {
+    userData: UserProfile;
+}
+
+const ExtraInfo: FC<ExtraInfoProps> = ({ userData }) => {
+    const handleVisitPage = () => {
+        if (typeof window !== 'undefined') {
+            window.location.href = "https://www." + userData.linkedIn;
+        }
+    };
```

Comment:
Let's make this more robust sooner that later, we can be certain that people will be posting LinkedIn URLs of varying formats and it's be good to test out during the demo

Something like this:
```
const handleVisitPage = () => {
    if (typeof window !== 'undefined') {
        // Normalize the LinkedIn URL
        let linkedInUrl = userData.linkedIn.trim();

        // Ensure the URL starts with "https://"
        if (!/^https?:\/\//i.test(linkedInUrl)) {
            linkedInUrl = "https://" + linkedInUrl;
        }

        // Redirect to the normalized URL
        window.location.href = linkedInUrl;
    }
};
```

Reply:
Sounds good 👍 Would we need to do any other processing to ensure the link provided was actually a linkedin link and not any other potentially harmful websites? Should I add a portion to ensure that the link given is actually a linkedin link?